<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book</class>
 <widget class="QMainWindow" name="Book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод данных</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название книги</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия автора</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Страна происхождения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Год выпуска</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Жанр</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_release">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_genre">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Англоязычная литература</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Детектив</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Детская литература</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Зарубежная литература</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Издание для досуга</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>История</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Компьютерная литература</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кулинарная литература</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Поэзия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Приключения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Проза</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Русская классика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ужасы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Фантастика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эротика</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_country">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Белоруссия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Великобритания</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>СССР</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Германия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Италия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Россия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>США</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Франция</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_publishing">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Издательство</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя автора</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
